settings.example.py

```diff
@@ -16,10 +16,3 @@
 
 # port for listening network socket
 PORT=12345
-
-# MQTT enable flag
-MQTT = True
-MQTT_HOST = ""
-MQTT_PORT = 123
-MQTT_USER = ""
-MQTT_PASS = ""
```

Comment:
Please squash/fixup this into the other commit

Reply:
squashed